exports/ceeb_complaint_export.sql

```diff
@@ -51,6 +45,6 @@ left join
 right join 
 	allegation_complaint ac on ac.complaint_identifier = cmp.complaint_identifier 
 where
-	cmp.incident_reported_utc_timestmp >= CURRENT_DATE - INTERVAL '1 year'
+	cmp.owned_by_agency_code = 'EPO'
```

Comment:
will this need to consider the referral history in the future? Do we have a parks story to cover this?